### ISSUE TITLE
Google Sign In: Sign Up Step 1: Auto-populate Fields with Google Info?

### ISSUE THREAD

**Issue by esteels** [ji-devs/ji-cloud#1035]
**Describe the bug**
Zeplin sketches suggest the login fields should populate Sign up Step 1 with suggestions: https://app.zeplin.io/project/5f10b1dc713f249d469be791/screen/5f6234d22ecbf8b9ab5800cb 
This does not happen.

**To Reproduce**
1. Go to Sign up https://jicloud.org/user/register
2. Select google account that has not been used before
3. See blank fields / no suggestions for the fields 

**Expected behavior**
I expect the fields to be auto-populated with suggestions from the google account per Zeplin screen.

**Screenshots**
NA

**Additional context**
Probably low priority.

**Comment by dakom** [comment]
Waiting on https://github.com/ji-devs/ji-cloud/issues/1163
